Codecademy/gamut
# feat(EditableIcon): Add SparkleIcon, BugIcon, ThumbsUpFilledIcon, ThumbsDownFilledIcon, and OpenAIFullIcon

### Overview



Adds SparkleIcon, BugIcon, ThumbsUpFilledIcon, ThumbsDownFilledIcon, and OpenAIFullIcon.



### PR Checklist

- [x] Related to designs: https://www.figma.com/file/skDEunydk4AfvXX40kXV6a/Dynamic-Hints?type=design&node-id=1548-2644&mode=design&t=WGSb625wWw3bLVYx-0
- [x] Related to JIRA ticket: [LEGO-3539](https://codecademy.atlassian.net/browse/LEGO-3539)

#### Testing Instructions

Ensure that all five icons are appearing in Storybook: https://653fdcd8d5f60200774b7616--gamut-preview.netlify.app/?path=/docs/atoms-icons-regular--editable-icon and check the linked preview environments.

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/36676)  | [Monolith Env](https://pr-36676-monolith.dev-eks.codecademy.com/learn) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/3942)  | [Portal Env](https://tayra.codecademy.com/projects/practice/honey-production?PR_ENV=le-pr-3942)   |




[LEGO-3539]: https://codecademy.atlassian.net/browse/LEGO-3539?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
